corehq/apps/es/es_query.py

```diff
@@ -71,6 +71,20 @@
 These things should nearly always be excluded, but if necessary, you can remove
 these with ``remove_default_filters``.
 
+Running against production
+--------------------------
+Since the ESQuery library is read-only, it's mostly safe to run against
+production. You can define alternate elasticsearch hosts in your localsettings
+file using the prefix 'ELASTICSEARCH_HOST_' and pass in this host name as the
+``debug_host`` to the constructor:
+
+.. code-block:: python
+
+    >>> print settings.ELASTICSEARCH_HOST_PROD
+    hqes0.internal-va.commcarehq.org
+    >>> CaseES(debug_host='prod').domain('dimagi').count()
+    120
```

Comment:
I'm happy to consider other interfaces if you've got suggestions.  I thought about:
* Require passing in the whole host URL to the constructor instead of an alias defined in localsettings (but that's tedious)
* Control which env to query solely based on a localsetting (that has some advantages, but is more magic than I'd like)